vagrant/commit_and_deploy.sh

```diff
@@ -14,3 +14,4 @@ fi
 
 eval $cmd
 ostree admin deploy vmcheck
+sync
```

Comment:
I think this is systemd getting a bit confused as to the unmount ordering.  Something to investigate later.